tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/tests/PitclipsePitSummaryViewTest.java

```diff
@@ -0,0 +1,66 @@
+package org.pitest.pitclipse.ui.tests;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.endsWith;
+import static org.hamcrest.Matchers.equalTo;
+import static org.pitest.pitclipse.ui.behaviours.pageobjects.PageObjects.PAGES;
+import static org.pitest.pitclipse.ui.view.PitView.BLANK_PAGE;
+
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.swtbot.swt.finder.junit.SWTBotJunit4ClassRunner;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.pitest.pitclipse.ui.behaviours.pageobjects.PitSummaryView;
+import org.pitest.pitclipse.ui.view.PitView;
+
+@RunWith(SWTBotJunit4ClassRunner.class)
+public class PitclipsePitSummaryViewTest extends AbstractPitclipseSWTBotTest {
+    private static final String TEST_PROJECT = "org.pitest.pitclipse.testprojects.twoclasses";
+    private static final String FOO_BAR_PACKAGE = "foo.bar";
+    private static final String FOO_BAR_PACKAGE_RESULT = "foo.bar";
+    private static final String FOO_CLASS_HEADER = "Foo.java";
+    private static final String FOO_CLASS_RESULT_URL_END = FOO_CLASS_HEADER + ".html";
+    private static final String INDEX_URL_END = "index.html";
+    private static final String INDEX_HEADER = "Pit Test Coverage Report";
+    private PitSummaryView summaryView;
+
+    @BeforeClass
+    public static void setupJavaProject() throws CoreException {
+        importTestProject(TEST_PROJECT);
+    }
+
+    @Before
+    public void resetView() throws InterruptedException {
+        // If this class is run alone, the view can't be closed and a RuntimeException
+        // is thrown as expected
```

Comment:
I'm not sure I understand this comment: do you mean that this test cannot be run alone?

Reply:
Is gone now, was connected with `closeViewById()`